test-utils/fixtures/messages/sections/dispatch/DispatchWarehouseExciseMessages.scala

```diff
@@ -26,6 +26,10 @@ object DispatchWarehouseExciseMessages {
     val hintText = "This is sometimes called an excise registration number (ERN), starting with GB or XI. For example, GB00123456789."
     val cyaLabel = "Excise ID (ERN)"
     val cyaChangeHidden = "Excise ID (ERN) of the tax warehouse of dispatch"
+
+    val exciseInvalid = "The excise ID for the tax warehouse of dispatch is not valid"
+    val exciseInvalid2 = "The excise ID for the tax warehouse of dispatch is not valid"
```

Comment:
Change this (exciseInvalid2) if the object name (in SubmissionFailureErrorCodes) changes